bench/dune

```diff
@@ -0,0 +1,5 @@
+(executable
+ (name main)
+ (libraries lockfree unix yojson)
```

Comment:
You may need to update the opam file with the deps.

Reply:
Thanks, tried that just now. Sadly, neither of ppx extensions seem to be available on 5.1.0: 

```    
- (invariant) → ocaml-variants >= 5.1.0+trunk → ocaml >= 5.1.0
- lockfree >= dev → ppx_deriving_yojson >= 3.6.1 → ppx_deriving >= 5.1 → ocaml < 4.12
- lockfree >= dev → ppx_yojson_conv >= v0.15.0 → ppxlib >= 0.23.0 → ocaml < 4.14
```

I'll write manual functions to generate json. I've tried just copy-pasting generated for `[@@deriving of_yojson]` but its a lot of code.